File: src/Services/InventoryService/InventoryService.Application/DTOs/Request/InventoryItem/CreateInventoryItemDto.cs
using Microsoft.AspNetCore.Http;

namespace InventoryService.Application.DTOs.Request.InventoryItem;

public class CreateInventoryItemDto
{
    public string Name { get; set; }
    public string UniqueCode { get; set; }
    public long Quantity { get; set; }
    public decimal EstimatedValue { get; set; }
    public DateTime ExpirationDate { get; set; }
    public Guid SupplierId { get; set; }
    public Guid WarehouseId { get; set; }
    public DateTime DeliveryDate { get; set; }
    public IFormFile DocumentFile { get; set; }
}

File: src/Client/inventory-track/src/app/models/dto/inventory-item/create-inventory-item-dto.ts
export class CreateInventoryItemDto {
  name!: string;
  uniqueCode!: string;
  quantity!: number;
  estimatedValue!: number;
  expirationDate!: string;
  supplierId!: string;
  warehouseId!: string;
  deliveryDate!: string;
  documentId!: string;

  // 🆕 Новые поля для ТТН
  batchNumber?: string;
  measureUnit?: string;
  vatRate?: number;
  placesCount?: number;
  cargoWeight?: number;
  notes?: string;
}
